packages/user-group-service/src/helpers.ts

```diff
@@ -83,6 +84,21 @@ class UserGroupApiHelper {
           .exec();
       } );
   }
+  // Helper function for rover interaction
+  public roverFetch( urlPart: String ) {
+    let credentials = `${ process.env.ROVER_USERNAME }:${ process.env.ROVER_PASSWORD }`;
+    return fetch(
+        `${ process.env.ROVER_API }${urlPart}`,
+        {
+          method: "GET",
+          headers: {
+            "Content-Type": "application/json",
+            "Authorization": `Basic ${ Buffer.from( credentials ).toString( 'base64' ) }`
+          },
+        }
+      ).
+      then( res => res.json() )
```

Comment:
```suggestion
      then( res => res.json() )
      
```

try to optimize the result.result variable here. so you can avoid the repetition from rover response.